Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -1019,6 +1010,25 @@ def update_remote_system_command(client, args):
         return remote_args.remote_incident_id
 
 
+def update_related_alerts(client: Client, args: dict):
+    new_status = args.get('status')
+    incident_id = args.get('incident_id')
+    comment = f"resolved by XSOAR, due to incident {incident_id} has been resolved"
+    demisto.debug(f"{new_status=}, {comment=}")
+    if not new_status:
+        raise DemistoException(f"Failed to update alerts related to incident {incident_id},"
+                               "no status found")
+    if new_status == 'resolved_duplicate_incident':
+        new_status = 'resolved_duplicate'
```

Comment:
Is there any other validation on this value? What if we send a value that doesn't exist in XDR?